client/components/jetpack/card/jetpack-product-card-alt-2/features-product-slide-out.tsx

```diff
@@ -43,15 +44,23 @@ const FeaturesProductSlideOut: FunctionComponent< Props > = ( {
 	);
 	const isDiscounted = isFinite( discountedPrice );
 	const productPrice = isDiscounted ? discountedPrice : originalPrice;
+	const formattedPrice = formatCurrency(
+		( ( product && product.displayPrice ) || productPrice ) as number,
+		( ( product && product.displayCurrency ) || currencyCode ) as string,
+		{
+			precision: 0,
+		}
+	);
+	const billingTimeFrame = durationToText(
+		( product && product.displayTerm ) || productBillingTerm
+	);
 
-	const billingTimeFrame = durationToText( productBillingTerm );
-
-	const slideOutButtonLabel = translate( 'Get {{name/}} $%(price)s', {
+	const slideOutButtonLabel = translate( 'Get {{name/}} %(price)s', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.